src/pullRequest.ts

```diff
@@ -0,0 +1,34 @@
+import {context, getOctokit} from '@actions/github'
+import * as core from '@actions/core'
+
+export async function getPullRequests() {
+  const token = core.getInput('ghToken')
+
+  if (token === '') {
+    return []
+  }
+
+  const repoOwner = context.repo.owner
+  const repo = context.repo.repo
+
+  const octokit = getOctokit(token)
+
+  try {
+    const {data: pullRequests} = await octokit.request(
+      'GET /repos/{owner}/{repo}/pulls',
+      {
+        owner: repoOwner,
+        repo,
+        headers: {
+          'X-GitHub-Api-Version': '2022-11-28'
+        }
+      }
+    )
+    return pullRequests
+  } catch (error) {
+    if (error instanceof Error) {
+      core.setFailed(error.message)
```

Comment:
rajoute ça pour avoir plus de details stp (regulierement le log simple n'affiche pas les details de l'appel API ou quoi)
```suggestion
      console.error(JSON.stringify(error))
      core.setFailed(error.message)
```